docs/changelog.rst

```diff
@@ -1,8 +1,37 @@
 =========
 Changelog
 =========
-
 * :release:`0.100.3-rc5 <2019-05-08>`
+* :bug:`4080` Fix "Unexpected signature format" issue in matrix transport
+* :feature:`4043` Update raiden-contracts to 0.19.0 with Görli Testnet support
+* :bug:`4041` Fix V20 -> V21 migration chain-state transformation to canonical identifiers
+* :bug:`4024` Fix clearing a channel state if and only if all unlocks are done by channel participants
+* :feature:`4002` Nightly builds are now deployed to Docker Hub
+* :bug:`3960` Fix KeyError for canonical identifier during Raiden restart
+* :bug:`3874` Fix invalidation of a batch-unlock transaction in case a similar transaction was already sent/mined by channel partner
+* :bug:`3856` Handle pruned blocks during settle/unlock when requesting on-chain state, use latest if block is pruned
+* :bug:`3850` Fix IOU request from PFS caused by not sending a complete payload as the endpoint requires
+* :feature:`3482` Provide ARM builds during CI deployments
```

Comment:
This is not something related to the changes contained in the new version, so should not be in the changelog.